ragna/deploy/_ui/modal_configuration.py

```diff
@@ -224,19 +221,19 @@ def advanced_config_ui(self):
                         self.config.param.chunk_size,
                         name="Chunk Size",
                         bar_color=ui.MAIN_COLOR,
-                        stylesheets=[ui.SS_LABEL_STYLE],
+                        css_classes=["modal_configuration_int_slider"]
+                        + (["disabled"] if disabled_source_storage else []),
                         width_policy="max",
                         disabled=disabled_source_storage,
-                        css_classes=["disabled"] if disabled_source_storage else [],
                     ),
                     pn.widgets.IntSlider.from_param(
                         self.config.param.chunk_overlap,
                         name="Chunk Overlap",
                         bar_color=ui.MAIN_COLOR,
-                        stylesheets=[ui.SS_LABEL_STYLE],
+                        css_classes=["modal_configuration_int_slider"]
```

Comment:
Same as above. Since it has two more occurrences below, would it make sense to define a 

```python
maybe_disabled = ["disabled"] if disabled_source_storage else []
```

on top and just use it here?